conjure-java-core/src/main/java/com/palantir/conjure/java/services/UndertowServiceHandlerGenerator.java

```diff
@@ -768,4 +774,13 @@ private static String deserializeFunctionName(Type type) {
                 .filter(arg -> arg.getParamType().accept(ParameterTypeVisitor.IS_BODY))
                 .collect(MoreCollectors.toOptional());
     }
+
+    private static String sanitizeVarName(String input, EndpointDefinition endpoint) {
+        String value = JavaNameSanitizer.sanitizeParameterName(input, endpoint);
+        if (RESERVED_PARAM_NAMES.contains(value)
+                || (endpoint.getReturns().isPresent() && RESULT_VAR_NAME.equals(value))) {
```

Comment:
I would just add result to the RESERVED_PARAM_NAMES to remove complexity at the cost of over-sanitizing in some cases.